README.md

```diff
@@ -24,16 +59,32 @@ import Button from '@e-toast/lib/Button';
 
```

Comment:
The usage section needs to be updated. I believe single component imports now become `import { Button } from '@e-toast/lib/components/Button'`. Please check it by running a build and checking the lib folder matches the expected structure.

Reply:
Done!